src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -196,26 +200,25 @@ class NexusPublishPlugin : Plugin<Project> {
             nexusRepo.snapshotRepositoryUrl.get()
         }
 
-    // For compatibility with 4.10.x (same as built-in `named<T>(String)` extension function)
-    private inline fun <reified T : Task> TaskContainer.withName(name: String): TaskProvider<T> {
-        return withType<T>().named(name)
-    }
-
-    private fun createSimplifiedCloseAndReleaseTask(rootProject: Project, extension: NexusPublishExtension) {
+    private fun configureSimplifiedCloseAndReleaseTask(rootProject: Project, extension: NexusPublishExtension) {
+        val closeAndReleaseSimplifiedTask = rootProject.tasks.named(SIMPLIFIED_CLOSE_AND_RELEASE_TASK_NAME)
         if (extension.repositories.isNotEmpty()) {
-            val closeAndReleaseSimplifiedTask = rootProject.tasks.register<Task>(SIMPLIFIED_CLOSE_AND_RELEASE_TASK_NAME) {
+            closeAndReleaseSimplifiedTask.configure {
                 val repositoryNamesAsString = extension.repositories.joinToString(",") { "'${it.name}'" }
                 val instanceCardinalityAwareString = if (extension.repositories.size > 1) { "instances" } else { "instance" }
                 description = "Closes and releases open staging repository in $repositoryNamesAsString Nexus $instanceCardinalityAwareString."
-                group = PublishingPlugin.PUBLISH_TASK_GROUP
             }
             extension.repositories.all {
                 val repositoryCapitalizedName = this.capitalizedName
-                val closeAndReleaseTask = rootProject.tasks.withName<Task>("closeAndRelease${repositoryCapitalizedName}StagingRepository")
+                val closeAndReleaseTask = rootProject.tasks.named<Task>("closeAndRelease${repositoryCapitalizedName}StagingRepository")
                 closeAndReleaseSimplifiedTask.configure {
                     dependsOn(closeAndReleaseTask)
                 }
             }
+        } else {
+            closeAndReleaseSimplifiedTask.configure {
+                didWork = false
```

Comment:
@marcphilipp The task is reported as:
> Task :closeAndReleaseStagingRepository UP-TO-DATE

instead of SKIPPED. Is that ok or `didWork` has to be configured in some other way?

Reply:
I guess `didWork` has to be set at the runtime.

Reply:
I was thinking about `doLast {}`, but that triggers some execution (instead of up-to-date).

Reply:
And as I tested it is still UP-TO-DATE :-/

Reply:
But maybe it works that way, anyway :)
```
> Task :closeAndReleaseStagingRepository UP-TO-DATE
Caching disabled for task ':closeAndReleaseStagingRepository' because:
  Build cache is disabled
Task ':closeAndReleaseStagingRepository' is not up-to-date because:
  Task has not declared any outputs despite executing actions.
:closeAndReleaseStagingRepository (Thread[Execution worker for ':',5,main]) completed. Took 0.001 secs.
```

Reply:
Without `didWork = false`:
```
> Task :closeAndReleaseStagingRepository UP-TO-DATE
Skipping task ':closeAndReleaseStagingRepository' as it has no actions.
:closeAndReleaseStagingRepository (Thread[Execution worker for ':',5,main]) completed. Took 0.0 secs.
```

Reply:
Hmm, we could actually set `isEnabled` to `false` initially and to `true` when we configure it for at least one staging repo.

Reply:
`isEnabled` works fine (although the output refers to `onlyIf` :) ):
```
> Task :closeAndReleaseStagingRepository SKIPPED
Skipping task ':closeAndReleaseStagingRepository' as task onlyIf is false.
:closeAndReleaseStagingRepository (Thread[Execution worker for ':',5,main]) completed. Took 0.001 secs.
```